res/css/views/dialogs/_InviteDialog.scss

```diff
@@ -195,6 +195,7 @@ limitations under the License.
     .mx_InviteDialog_roomTile_nameStack {
         display: inline-block;
         overflow: hidden;
+        width: calc(70%);
```

Comment:
why 70%?

Reply:
It was chosen so that the layout remained intact, because if it was set to 100% the name would appear under the users avatar like in the first picture, it would also push the "23 hours ago" underneath the box. Setting it to 70% fixed those issues.

Reply:
By setting so, the time strings can be placed under the nameStack due to float and/or text-align stuff.

![before](https://user-images.githubusercontent.com/3362943/160256744-5d685850-cbca-43df-91d7-0b196217f9dd.png)

It looks like the display style of roomTile in which they are placed needs changing with flex/grid. From my trial I guess that should be the most straightforward way.

Reply:
The issue with 70% is that the UI cuts off prematurely in the screenshots, so I suspect the change is a bit more involved than what is implied here.

Also, magic numbers like these need comments in the code.